.circleci/src/workflows/deliver_beta_administration.yml

```diff
@@ -0,0 +1,22 @@
+when: << pipeline.parameters.run_deliver_beta_administration >>
+jobs:
+  - bump_version:
+        prepare_delivery: false
+        context:
```

Comment:
prepare_delivery should be true here, shouldn't it?

Reply:
the problem here is @ztefanie if we set the this flag for the beta releases the git tags will be created.
If we then trigger the production release, we can not create the same tags again. 
we could add `-beta` to the tags or just create the tag on production

Reply:
Why not do it as we do it for Integreat:
Deploy beta -> Create github tags
Promote beta to production -> Create no tags
Deploy live directly (we rarly do this) -> Create tag

What would the problem be, if we have a github tag, but the version is only available on beta?

Reply:
ok then i keep deploying directly live